faq/manage-cluster-faq.md

```diff
@@ -111,6 +111,58 @@ TiDB 目前社区非常活跃，同时，我们还在不断的优化和修改 BU
 - 如果开启了 TiDB Binlog，受 TiDB Binlog 的实现原理限制，TiDB 不会使用异步提交或一阶段提交特性。
 - TiDB 只在事务写入不超过 256 个键值对，以及所有键值对里键的总大小不超过 4 KB 时，才会使用异步提交或一阶段提交特性。这是因为对于写入量大的事务，异步提交不能明显提升执行性能。
 
+### TiUP 如何为已存在的 TiDB 集群开启组件加密？
+
+TiUP v1.9.0 版本开始支持开启组件加密功能，下面以**已存在但并未开启组件间加密的集群**为例，为其开启组件加密，**该操作不支持在线进行需要重启集群**，步骤及说明如下：
+
+1. 通过 `tiup cluster tls {cluster_name} enable/disable` 命令开启或关闭 TLS 组件加密。首先，查看集群初始状态：
+
+    ```shell
+    tiup cluster display {cluster_name}
+    ```
+
+2. 通过 edit-config 将 pd 节点信息保存到单独文件，方便后续扩容。由于 ETCD 机制限制，需先将 PD 节点缩容到 1 个节点后，再使用 `tls ${cluster_name} enable` 开启组件加密。注意：`tls ${cluster_name} enable/disable` 方式开启或关闭 TLS 时会重启 TiDB 集群，行为类似于 tiup cluster restart ，而不是以滚动重启的方式重启节点。
+
+3. 将 PD 节点缩容至单节点
+
+    ```shell
+    tiup cluster scale-in {cluster_name} -N {PD_IP:PD_Port}
+    tiup cluster display {cluster_name}
+    ```
```

Comment:
````suggestion
1. 查看集群初始状态，并通过 `tiup cluster edit-config` 备份 PD 节点信息，以便后续扩容：

    ```shell
    tiup cluster display ${cluster_name}
    tiup cluster edit-config ${cluster_name}
    ```

2. 将 PD 节点缩容至单个节点：

    ```shell
    tiup cluster scale-in ${cluster_name} -N ${PD_IP:PD_Port}
    tiup cluster display ${cluster_name}
    ```
````